docs/book/src/networking-introduction.md

```diff
@@ -0,0 +1,82 @@
+
+## 1. TCP/IP and UDP Comparison
+
+Let's compare TCP, UDP, and QUIC.
+
+- **unreliable**: Transport packeten zijn er niet van verzekerd dat ze aan komen bij hun eindbestemming en op volgorde zijn. 
+- **reliable**: Transport packeten zijn er van verzekerd dat ze op volgorde op hun eindbestemming komen.
+
+Unreliability gives great uncertainty with a lot of freedom, while reliability gives great certainty with costs in speed and freedom.
+That is why some protocols such as QUIC, SCTP are built on UDP instead of TCP. 
+
+| Feature |  TCP  | UDP | QUIC
+| :-------------: | :-------------: | :-------------: | :-------------: |
+|  [Connection-Oriented][6]           |       Yes         | No                       | Yes
+|  Transport Guarantees               | Reliable          | Unreliable               | Reliable and/or unreliable with extension 
+|  Packet Transfer                    | [Stream-based][4] | Message based            | Message based and/or Stream based
+|  Header Size                        |  20 bytes         | 8 bytes                  |  ~16 bytes(depending on connection id)  
+|  [Control Flow, Congestion Avoidance/Control][5] | Yes  | No                       |  ** Yes, and possible controlled by userspace                                          
+|  Based On | [IP][3]                 | [IP][3]           |  UDP
+
+** QUIC control flow/congestion implementations will run in userspace wereas in TCP its running in kernelspace, 
+however there might be a kernel implementations for QUIC in the future.
+
+## 2. Issues with TCP 
+
+TCP has been around for a long time and was not designed with the modern internet in mind. 
+It has several difficulties that QUIC tries to resolve. 
+
+### Head-of-line Blocking
+
+One of the biggest issues with TCP is that of Head-of-line blocking. 
+It is a convenient feature because it ensures that all packages are sent and arrive in order. 
+However, in cases of high throughput (multiplayer game networking) and big load in a short time (web page load), this can severely impact latency.
+
+The issue is demonstrated in the following animation:
+
+![Head of line blocking][animation] 
+
+This animation shows that if a certain packet drops in transmission, all packets have to wait at the transport layer until it is resent by the other end.
+If the dropped packet is resent and arrived then all packets are freed from the transport layer. 
+
+Let's look at two areas where this head-of-line blocking issue creates issues. 
+
+**Web Networking**
+
+The last years websites have been growing in size which causes. 
+This increases the loading time and makes head-of-line blocking a more concerning topic. 
+To tackle this issue HTTP-2 has introduced a technique called multiplexing. 
+In short, this means that multiple TCP streams are initialized to communicate with the server. 
+It allows a server to transfer multiple sources in parallel over a single stream.
```

Comment:
As websites increasingly need a larger number of HTTP requests (HTML, CSS, JavaScript, images) to display all content, the impact of head-of-line blocking has also increased. To improve on this, HTTP 2 introduced request multiplexing within a TCP data stream, which allows servers to stream multiple responses at the same time. However, data loss of a single packet will still block all response streams because they exist within the context of a single TCP stream.